### PR TITLE
Release v0.1.1

### PR DESCRIPTION
**Pull request by hugobarauna**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.2.0](https://github.com/livebook-dev/kino_slack/tree/v0.1.0) (2023-03-XX)
```
Comment (jonatanklosko) [inline CHANGELOG.md]:
```suggestion
## [v0.1.1](https://github.com/livebook-dev/kino_slack/tree/v0.1.1) (2023-03-17)
```

File: guides/components.livemd [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 ```elixir
 Mix.install([
-  {:kino_slack, "~> 0.1.0"},
+  {:kino_slack, "~> 0.2.0"},
```
Comment (jonatanklosko) [inline guides/components.livemd]:
```suggestion
  {:kino_slack, "~> 0.1.1"},
```

**Review by jonatanklosko — APPROVED** [review]
Let's update the version in mix.exs :)

**Comment by hugobarauna** [comment]
@jonatanklosko is everything ok, now?

**Comment by jonatanklosko** [comment]
@hugobarauna ship it!

**Pull request merged**